storage/notifications.go

```diff
@@ -104,18 +104,6 @@ func toNotificationFromProto(pbn *storagepb.NotificationConfig) *Notification {
 	return n
 }
 
```

Comment:
Add notes to the exported methods in this file indicating that gRPC is not supported.